CMakeLists.txt

```diff
@@ -3023,7 +3095,7 @@ if(SDL_GPU)
     set(SDL_GPU_D3D11 1)
     set(HAVE_SDL_GPU TRUE)
   endif()
-  if(SDL_RENDER_D3D12)
+  if(WINDOWS)
```

Comment:
Did you rebase on top of the latest sdl3 master?
These changes are already in git.

Reply:
I believe this is resolved?

Reply:
Yup :)